ext/hal/nxp/imx/utilities/inc/debug_console_imx.h

```diff
@@ -0,0 +1,154 @@
+/*
```

Comment:
Is there any point importing utilities/ since I'm not sure if we'd ever use debug_console.